### PR TITLE
Add doc about tilegeneration UI

### PR DESCRIPTION
**Pull request by asaunier**

### CODE REVIEW

File: doc/integrator/tilegeneration.rst.tmpl [hunk 1]
```diff
@@ -126,3 +126,13 @@ code::
 
   export AWS_ACCESS_KEY_ID=<access_key_id>
   export AWS_SECRET_ACCESS_KEY=<secret_access_key>
+
+Web Interface
+~~~~~~~~~~~~~
+
+It is possible to run tile generation commands through a web interface located at URL
+``<application main URL>/tiles/admin/``. This interface is protected by a password that
+must be specified in environnement variable C2C_SECRET in file `env.default`.
```
Comment (julsbreakdown) [inline doc/integrator/tilegeneration.rst.tmpl:135]: "in the environnement "
"in the file"

Comment (asaunier) [reply doc/integrator/tilegeneration.rst.tmpl:135]: Even with a "proper" name right after?

Comment (sbrunner) [reply doc/integrator/tilegeneration.rst.tmpl:135]: The env.default should not be modified, then it should be moved into the env.project file.

Comment (asaunier) [reply doc/integrator/tilegeneration.rst.tmpl:135]: @sbrunner I have found the mention to ``env.default`` in https://github.com/camptocamp/c2cgeoportal/blob/master/doc/integrator/c2cwsgiutils.rst
Should I update it as well?

Comment (asaunier) [reply doc/integrator/tilegeneration.rst.tmpl]: @sbrunner I have update the filename as you suggested, as well as a few line above and in the c2cwsgiutils doc. See https://github.com/camptocamp/c2cgeoportal/pull/6165/commits/eb972cf854e89d8a935cbd84d25d1465eac03560

@julsbreakdown I have inverted the words order and added some "the". Is it correct?

Thanks!

File: doc/integrator/tilegeneration.rst.tmpl [hunk 2]
```diff
@@ -126,3 +126,13 @@ code::
 
   export AWS_ACCESS_KEY_ID=<access_key_id>
   export AWS_SECRET_ACCESS_KEY=<secret_access_key>
+
+Web Interface
+~~~~~~~~~~~~~
+
+It is possible to run tile generation commands through a web interface located at URL
+``<application main URL>/tiles/admin/``. This interface is protected by a password that
+must be specified in the ``C2C_SECRET`` environnement variable in the ``env.project`` file.
```
Comment (jwkaltz) [inline doc/integrator/tilegeneration.rst.tmpl:135]: environnement -> environment

Comment (asaunier) [reply doc/integrator/tilegeneration.rst.tmpl]: @jwkaltz Thanks! Updated

**Comment by asaunier** [comment]
I have squashed the fixup commits.
@sbrunner Feel free to merge the PR if it looks good to you.

File: doc/integrator/tilegeneration.rst.tmpl (hunk 1 above)
Comment (sbrunner) [reply doc/integrator/tilegeneration.rst.tmpl]: > Should I update it as well?

Yes thanks :-)

**Pull request merged**